test/TEST-50-MULTINIC/test.sh

```diff
@@ -219,28 +219,30 @@ test_setup() {
         [ -f /etc/netconfig ] && inst_multiple /etc/netconfig
         type -P dhcpd > /dev/null && inst_multiple dhcpd
         [ -x /usr/sbin/dhcpd3 ] && inst /usr/sbin/dhcpd3 /usr/sbin/dhcpd
-        instmods nfsd sunrpc ipv6 lockd af_packet
+        instmods nfsd sunrpc ipv6 lockd af_packet 8021q ipvlan macvlan
         inst_simple /etc/os-release
         inst ./server-init.sh /sbin/init
         inst ./hosts /etc/hosts
         inst ./exports /etc/exports
         inst ./dhcpd.conf /etc/dhcpd.conf
-        inst_multiple /etc/nsswitch.conf /etc/rpc /etc/protocols
+        inst_multiple -o {,/usr}/etc/nsswitch.conf {,/usr}/etc/rpc {,/usr}/etc/protocols
 
         inst_multiple rpc.idmapd /etc/idmapd.conf
 
         inst_libdir_file 'libnfsidmap_nsswitch.so*'
         inst_libdir_file 'libnfsidmap/*.so*'
         inst_libdir_file 'libnfsidmap*.so*'
 
-        _nsslibs=$(sed -e '/^#/d' -e 's/^.*://' -e 's/\[NOTFOUND=return\]//' /etc/nsswitch.conf \
-            | tr -s '[:space:]' '\n' | sort -u | tr -s '[:space:]' '|')
+        _nsslibs=$(
```

Comment:
It's always better ( and resource cheaper ) to include our own file as opposed to be sed-ing existing ones and it does not scale since one distribution does one thing and another distribution does another.

Reply:
Including more libraries in a test root is always better. We don't want failing stuff, because a library was missing, which needs to be dlopen'ed. This does no harm, other than disk space. It's not changing any config.

Reply:
Just a thought on that note maybe we can be a bit smarter about library inclusion in the future and create something like an binaries() section in which we list the binaries ( which always know since we usually add them as a requirement or in the install() section  ) which then would be parsed with something along the lines of

```
	for file in $(the-listed-binaries-in-binaries()); do \
		cp -fL $$file initramfs/$$file || exit 1;\
		if file $$file | grep -q ELF; then \
			for lddfile in $$(ldd $$file | awk '{print $$3}' | grep "^/"); do \
				file_path=$$(dirname $$lddfile); \
				cp -Ln $$lddfile initramfs/$$file_path/; \
			done \
		fi \
	done
```
And or passes the output to to inst_libdir_file instead.

Reply:
dracut-install does the `ldd` thing already. It's just for `dlopen`'ed libraries, which we can't catch.